qutebrowser/config/configdata.yml

```diff
@@ -2362,6 +2366,7 @@ bindings.default:
       <Ctrl-Return>: follow-selected -t
       .: repeat-command
       <Ctrl-p>: tab-pin
+      <Alt-m>: tab-mute
```

Comment:
Please watershed my choice of keybind, this is what I have on my system, so it's probably bad :P

Reply:
FWIW the only data point I found is the [Mute Tab Shortcuts](https://chrome.google.com/webstore/detail/mute-tab-shortcuts/opcjanmpjbdbdpnjfjbboacibokblbhl?hl=en) Chrome extension which uses Alt-Shift-m. I have no idea why though, and I'm fine with Alt-m.